InstrumentationEngine.sln

```diff
@@ -688,73 +688,73 @@ Global
 		{9F97B063-5ACF-409A-8DCC-4BD9FF817D1D}.Release|x86.ActiveCfg = Release|Any CPU
 		{9F97B063-5ACF-409A-8DCC-4BD9FF817D1D}.Release|x86.Build.0 = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Managed|Any CPU.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Managed|Any CPU.Build.0 = Release|Any CPU
-		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Managed|ARM64.ActiveCfg = Debug|ARM64
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Managed|ARM64.ActiveCfg = Debug|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Managed|x64.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Managed|x86.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Native|Any CPU.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Native|Any CPU.Build.0 = Release|Any CPU
-		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Native|ARM64.ActiveCfg = Debug|ARM64
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Native|ARM64.ActiveCfg = Debug|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Native|x64.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Native|x86.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|ARM64.ActiveCfg = Debug|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|x64.ActiveCfg = Debug|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|x64.Build.0 = Debug|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|x86.Build.0 = Debug|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|Any CPU.Build.0 = Release|Any CPU
-		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|ARM64.ActiveCfg = Release|ARM64
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|ARM64.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x64.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x64.Build.0 = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x86.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x86.Build.0 = Release|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|Any CPU.ActiveCfg = Release|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|Any CPU.Build.0 = Release|Any CPU
-		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|ARM64.ActiveCfg = Debug|ARM64
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|ARM64.ActiveCfg = Debug|Any CPU
```

Comment:
FYI, this looks like the correct change, but the project builds an x86 assembly so when you go to run it on ARM64, it _might_ fail.

Reply:
I'll make this explicitly x86